tools/monitoring/main.go

```diff
@@ -57,3 +76,40 @@ func hello(w http.ResponseWriter, r *http.Request) {
 	errorPatterns := yamlFile.CollectErrorPatterns()
 	fmt.Fprintf(w, "error patterns collected from yaml:%s", errorPatterns)
 }
+
+func testCloudSQLConn(w http.ResponseWriter, r *http.Request) {
+	log.Printf("Serving request: %s", r.URL.Path)
+	log.Println("Testing mysql database connection.")
+
+	conn, err := getConn(dbConfig)
+	if err != nil {
+		fmt.Fprintf(w, "Failed to ping database: %v\n", err)
+		return
+	}
+	defer conn.Close()
+
+	fmt.Fprintf(w, "Success\n")
+}
+
+func configureMonitoringDatabase(fUser string, fPass string) (DBConfig, error) {
+	var config DBConfig
+
+	user, err := ioutil.ReadFile(fUser)
+	if err != nil {
+		return config, err
+	}
+
+	pass, err := ioutil.ReadFile(fPass)
+	if err != nil {
+		return config, err
+	}
+
+	config = DBConfig{
+		Username:     string(user),
+		Password:     string(pass),
+		DatabaseName: os.Getenv("MONITORING_DB_NAME"),
```

Comment:
can we pass this as flags instead of env var?

Reply:
Done